openedx/core/djangoapps/discussions/docs/decisions/0006-discussions-api-fetch-inactive-providers.rst

```diff
@@ -0,0 +1,150 @@
+Allow fetching configuration for inactive providers
+===================================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+In [ADR 003](./0003-configuration-rest-api.rst) an API was proposed for
+discussion configuration. This proposed a single API that returned
+information about the active provider and listed other available providers.
+
+There are a few issues with this API that we'd like to rectify such as:
+
+- It includes a lot of information that is mostly read-only.
+- It doesn't provide any way to fetch setting for inactive provider.
+
+By moving the bits of the API that are read-only to a separate API we can
+cache the response better and that data doesn't need to be processed and
```

Comment:
I can read a word cache, is this something that exists today or it will be a new feature of the API? 

If its a new feature, can you add a little detail about what kind of cache you are thinking (caching response) or any details that can be helpful? thanks.

Reply:
Sure, my thinking is mainly that the frontend doesn't even need to call this API twice, so it can be cached there. However it can also be cached in the backend.

Reply:
Thanks, can we elaborate it in the ADR too?